halo/app/start.go

```diff
@@ -314,7 +314,12 @@ func newCometNode(ctx context.Context, cfg *cmtcfg.Config, app *App, privVal cmt
 		},
 	)
 
-	// Don't instantiate the default prometheus server.
+	// Configure CometBFT prometheus metrics as per provided config
+	metrics := node.DefaultMetricsProvider(&cmtcfg.InstrumentationConfig{
+		Prometheus: cfg.Instrumentation.Prometheus,
+		Namespace:  cfg.Instrumentation.Namespace,
```

Comment:
I think is that you were missing in your PR. That resulted in incorrect metric names so it seemed like it wasn't renabled.

Reply:
Yes that must be it because I saw there is no prefix `cometbft_` 👍

Thanks a lot Corver!